src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,98 @@
+package duke;
+
+
+import java.io.IOException;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+/**
+ * Parse user input.
+ */
+public class Parser {
+
+    protected static String[] parseTypingInput(String str) {
+        String allLowerCase = str.toLowerCase();
+        return allLowerCase.strip().split(" ", 2);
+    }
+
+    /**
+     * Parses user input.
+     */
+    public static void parseInput() {
+        Scanner sc = new Scanner(System.in);
+        boolean running = true;
```

Comment:
Perhaps it would be better if the variable name is renamed ti indicate that it is boolean.